lib/core/network/protocols/websocket.js

```diff
@@ -208,6 +208,12 @@ class WebSocketProtocol extends Protocol {
 
     try {
       parsed = JSON.parse(data);
+      if (parsed.p && parsed.p === 1) {
+        debug('[%s] sending back a "pong" message', connection.id);
+        wsConnection.alive = true;
+        this._send(connection.id, '{"p":1}');
```

Comment:
The PONG response needs to be different than the PING query:

```suggestion
        this._send(connection.id, '{"p":2}');
```